### DISCUSSION TITLE
Are my EF LINQ to SQL queries safe?

### DISCUSSION THREAD

**Discussion by giscus** [linkdotnet/Blog.Discussions#109] [General]
# Are my EF LINQ to SQL queries safe?

Have you ever asked yourself if your EF LINQ queries are susceptible to SQL injection attacks? Either because you are querying some user data from a text field or directly taking whatever your API hits against the database?

https://steven-giesel.com/blogPost/35ec8819-220a-42bc-9224-a812ec358434

**Comment by kapsiR** [comment]
I think it would be good to mention `FromSqlInterpolated` and `FromSql` here too 😉

**Reply by linkdotnet** [reply]
Good points - will add them.

**Reply by linkdotnet** [reply]
Said and done!
